### PR TITLE
feat: try multiple faxbots if the first fails in the CLI

### PR DESCRIPTION
**Pull request by midgleyc**
Reference: https://kolmafia.us/threads/trying-to-use-ash-to-faxbot-a-factory-overseer-fails.26271/

The bug occurs when, for example:

1. easyfax is tried first
2. easyfax offers the monster you want
3. easyfax has become trapped in a clan over rollover, so can't actually return the monster you want

The thread talks about several other options, but I think this change is still good.

### CODE REVIEW

File: src/net/sourceforge/kolmafia/textui/command/FaxbotCommand.java [hunk 1]
```diff
@@ -59,9 +61,13 @@ public void run( final String cmd, final String command )
 			}
 
 			Monster monster = bot.getMonsterByCommand( (String)commands.get( 0 ) );
-			FaxRequestFrame.requestFax( botName, monster, false );
-			return;
+			tried = true;
+			if (FaxRequestFrame.requestFax( botName, monster, false )) {
```
Comment (gausie) [inline src/net/sourceforge/kolmafia/textui/command/FaxbotCommand.java:65]: This needs mafia-style formatting

**Comment by gausie** [comment]
I can't remember, when does `requestFax` decide that a fax request has failed?

File: src/net/sourceforge/kolmafia/textui/command/FaxbotCommand.java (hunk 1 above)
Comment (midgleyc) [reply src/net/sourceforge/kolmafia/textui/command/FaxbotCommand.java:65]: do you have a styling file or lint I can use?

**Comment by midgleyc** [comment]
Good point, lots and lots of reasons:
1. can't receive a fax
2. bot not online
3. used to check whether user is in chat, but that code ignored by the `|| true`
4. user refused to dump their fax
5. bot didn't respond
6. bot said the fax wasn't available

Of those, I guess 2, 5 and 6 should try another bot.

**Comment by midgleyc** [comment]
Looks like ASH tries again for any failure.

File: src/net/sourceforge/kolmafia/textui/command/FaxbotCommand.java (hunk 1 above)
Comment (midgleyc) [reply src/net/sourceforge/kolmafia/textui/command/FaxbotCommand.java]: I found, and used, the one [posted by heeheehee](https://kolmafia.us/threads/mafia-source-code-formatting.26013/#post-161996).

**Comment by jaadams5** [comment]
I'm still trying to figure out how to do things so forgive me for not commenting on a line of unchanged code.  It seems to me that if I try to get a fax from one bot and there are too many matches then I really ought to try another bot rather than bail immediately.

**Review by jaadams5 — APPROVED** [review]

**Pull request merged**